src/components/UI/ToggleSwitch/index.tsx

```diff
@@ -1,44 +1,34 @@
-import classNames from 'classnames'
 import React, { ChangeEventHandler } from 'react'
 import './index.css'
 
 interface Props {
-  dataTestId?: string
+  htmlId: string
   disabled?: boolean
   handleChange: ChangeEventHandler<HTMLInputElement>
   value: boolean
   title: string
 }
 
 export default function ToggleSwitch(props: Props) {
-  const {
-    handleChange,
-    value,
-    disabled,
-    title,
-    dataTestId = 'toggleSwitch'
-  } = props
-  // TODO fixes errors in the console, but the props may not be necessary at all
-  const checkmarkProps = {
```

Comment:
I was able to remove the need of the `.checkmark` element completely so we don't need this anymore